src/styles/index.scss

```diff
@@ -2,2 +1,86 @@
+@import './variables';
+@import './card';
+@import './stars';
+
+html {
+  font-family: Roboto, Arial, sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+
+  --active-color: #00acdc;
+}
+
+.body {
+  width: 100%;
+  margin: 0;
+  padding: 0;
+}
+
+.header {
+  display: flex;
+  width: 100%;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.nav {
+  margin: 0 50px 0 0;
+}
+
+.nav__list {
+  display: flex;
+  margin: 0;
+  padding: 0;
+  list-style: none;
+}
+
+.nav__item {
+  display: block;
+  height: 60px;
+  line-height: 60px;
+
+  color: #060b35;
+  text-decoration: none;
+  text-transform: uppercase;
+}
+
+li {
+  margin-right: 20px;
+}
+
+li:last-child {
+  margin-right: 0;
+}
+
+.is-active::after {
+  position: absolute;
+  bottom: 0;
+  display: block;
+  content: '';
+  height: 4px;
+  width: 100%;
+  background-color: var(--active-color);
+  border-radius: 8px;
+}
```

Comment:
Since `.is-active` is used for a state, it should be combined with the element's class it's related to, e.g., `.nav__item.is-active`, to increase specificity and reduce the risk of unwanted style overrides.